coordinator/src/services/translation/demographics.ts

```diff
@@ -23,7 +23,6 @@ export function convertName(fhirHumanName: fhir.HumanName): core.Name {
 function convertNameUse(fhirNameUse: string) {
   switch (fhirNameUse) {
   case "usual":
-    return core.NameUse.USUAL
```

Comment:
This looks like a behaviour change rather than refactoring / code improvements, it would be better if this was in a separate PR